src/Microsoft.AspNetCore.Mvc.Core/Resources.resx

```diff
@@ -297,6 +297,9 @@
   <data name="ModelBinding_MissingBindRequiredMember" xml:space="preserve">
     <value>A value for the '{0}' property was not provided.</value>
   </data>
+    <data name="ModelBinding_MissingRequestBodyRequiredMember" xml:space="preserve">
+    <value>A value for the request body was not provided.</value>
```

Comment:
Suggest `A non-empty request body is required.` because "value" doesn't mean much here.